corehq/apps/cloudcare/templates/cloudcare/cloudcare_home.html

```diff
@@ -32,6 +32,8 @@
     <script language="javascript" type="text/javascript" src="{% static 'formplayer/script/entrycontrols_full.js' %}?version={{app_version}}"></script>
     <script language="javascript" type="text/javascript" src="{% static 'formplayer/script/build/fullform-ui.templates.js' %}?version={{app_version}}"></script>
     <script language="javascript" type="text/javascript" src="{% static 'formplayer/script/fullform-ui.js' %}?version={{app_version}}"></script>
+    <script language="javascript" type="text/javascript" src="{% static 'formplayer/script/vkbeautify.0.99.00.beta.js' %}"></script>
```

Comment:
think you could move these to the `vendor` folder? i'm trying to organize TF scripts a bit